components/minhash_generator/src/main.py

```diff
@@ -0,0 +1,58 @@
+"""A component that generates minhashes of text."""
+import logging
+
+import pandas as pd
+from datasketch import MinHash
+from fondant.component import PandasTransformComponent
+from nltk.util import ngrams
+
+logger = logging.getLogger(__name__)
+
+
+class MinHashGeneratorComponent(PandasTransformComponent):
+    """Component generates minhashes of text."""
+
+    @staticmethod
```

Comment:
can we move those methods above and only have the class implement the `transform`?  This is a bit similar to we have it for other components [example](https://github.com/ml6team/fondant/blob/main/components/image_cropping/src/main.py#L15), it makes the layout a bit nicer than having to write static methods